main.go

```diff
@@ -40,8 +41,6 @@ var (
 	vpcNetworkName             = flag.String("vpc-network-name", "default", "VPC network name")
 	localityZone               = flag.String("locality-zone", "", "the locality zone to use, instead of retrieving it from the metadata server. Useful when not running on GCP and/or for testing")
 	ignoreResourceDeletion     = flag.Bool("ignore-resource-deletion-experimental", false, "assume missing resources notify operators when using Traffic Director, as in gRFC A53. This is not currently the case. This flag is EXPERIMENTAL and may be changed or removed in a later release.")
-	includeV3Features          = flag.Bool("include-v3-features-experimental", true, "whether or not to generate configs which works with the xDS v3 implementation in TD. This flag is EXPERIMENTAL and may be changed or removed in a later release.")
```

Comment:
We still need this because `include-v3-features-experimental=False` enables xDS v2, which is still technically supported.

Reply:
v2 works with this config, if the client only supports v2.

Reply:
We spoke offline. We have some tests that test v2-and-v3-supporting clients work when v3 is not in the bootstrap file. That should just use an old bootstrap generator (from before we changed the default), as that's the only situation a customer would be in that situation. We'll freeze the bootstrap generator it is using for v2 until the v2 testing stuff is deleted outright. @sergiitk will make that change internally; it'll be pretty easy.

Reply:
ref cl/579998281